packages/openapi-generator/src/internal.ts

```diff
@@ -0,0 +1,10 @@
+export * from './file-serializer';
+export * from './file-writer';
+export * from './options';
+export * from './parser';
+export * from './sdk-metadata';
+export * from './document-converter';
+export * from './extensions';
+export * from './generator';
+export * from './openapi-types';
+export * from './schema-util';
```

Comment:
I know the `cli.ts` might not contain any internal "APIs", but the `internal.ts` should contain "all" source code. Could we all it as well?